### ISSUE TITLE
Allow mutation fill color to be configurable

### ISSUE THREAD

**Issue by cheapsteak** [oncojs/lolliplot#32]
Proposed API

```js
lolliplot({
  getMutationColor: function (d) { /*return a string */}
})
```

e.g.

```js
lolliplot({
  getMutationColor: (d) => ({high: '#f00', low:'#fff', unknown: '#777'})[d.impact]
})
```

**Comment by alex-wilmer** [comment]
I like this idea

**Comment by cheapsteak** [comment]
closed by #45
